classes/Kohana/Model/Purchase.php

```diff
@@ -9,6 +9,18 @@
  * @author     Ivan Kerin <ikerin@gmail.com>
  * @copyright  (c) 2013 OpenBuildings Ltd.
  * @license    http://spdx.org/licenses/BSD-3-Clause
+ *
+ * @property Jam_Field_Primary      $id
+ * @property Jam_Field_String       $currency
+ * @property Jam_Field_Serialized   $monetary
+ * @property Jam_Field_Boolean      $is_frozen
+ * @property Jam_Field_Timestamp    $created_at
+ * @property Jam_Field_Timestamp    $updated_at
```

Comment:
These magic properties above would not actually return the type of the field, but the scalar value behind it. E.g. string, int, array etc.

Reply:
Fixed so it returns the actual return types.